src/components/ItemTooltip/index.jsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import styled from 'styled-components';
+import items from 'dotaconstants/build/items.json';
+import constants from '../constants';
+
+const itemAbilities = {
+  active: {
+    color: '#4183d7',
+    headerColor: 'rgba(65, 131, 215, 0.25)',
+    text: 'Active',
```

Comment:
localize?

Reply:
but the rest of the tooltip is in english too. I thought it would be pointless

Reply:
yeah that's true.  up to you if you want to do this, it might make it easier if we one day localize the tooltip fully